pkg/adaptor/hypervisor/aws/service.go

```diff
@@ -104,6 +104,12 @@ func (s *hypervisorService) CreateVM(ctx context.Context, req *pb.CreateVMReques
 	if pod == "" {
 		return nil, fmt.Errorf("pod name %s is missing in annotations", annotations.SandboxName)
 	}
+	// cri-o stores the sandbox name in the form of k8s_<pod name>_<namespace>_<uid>_0
+	// Extract the pod name from it.
+	if tmp := strings.Split(pod, "_"); len(tmp) > 1 && tmp[0] == "k8s" {
+		pod = tmp[1]
+	}
+
```

Comment:
@yoheiueda possible to put this snippet in the utils package?

Reply:
I updated the code. Could you review again?